CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Rubrik CDM As Built Report Changelog
 
+### Modified
```

Comment:
Need to increase the version number to allow updates to be published to PS Gallery. You will also need to modify the version in the manifest.

Reply:
Done!